README.md

```diff
@@ -155,17 +154,15 @@ main states are:
 - Send: The state machine has received a mission and is sending the mission to the robot.
 - Monitor: The robot has received a mission from the state machine, and the state machine is monitoring the current
   mission.
-- Cancel: The state machine has received a request to abort, or an event has occurred which requires the mission to be
-  canceled. The cancel state also functions as a wrap-up state when a mission is finished, prior to the state machine
-  returning to idle.
+- Finalize: The state machine has finished the mission, received a request to abort, or an event has occurred which
+  requires the mission to be canceled. Once finished the state machine will return to idle.
```

Comment:
canceled -> finalized?

Reply:
Same as above. The wording is correct because we have three scenarios which lead to entering Finalize: A completed mission, a a request to abort or a cancelled mission.